stl/debugger/STL.natvis

```diff
@@ -1030,85 +1028,14 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </Expand>
   </Type>
 
-  <Type Name="std::basic_string&lt;wchar_t,*&gt;">
-    <AlternativeType Name="std::basic_string&lt;unsigned short,*&gt;" />

```

Comment:
I think we should restore this special case but only for `unsigned short` to preserve the `/Zc:wchar_t-` user experience. We've deprecated `/Zc:wchar_t-` a bit in the STL, but I see no reason to regress debugging for the poor folks who have to maintain ancient code that needs it.

Reply:
Comment retracted per discussion starting at https://github.com/microsoft/STL/pull/5177#discussion_r1878546856.